### ISSUE TITLE
Question: Is it intended that the dependencies of the root index.ts are not rendered?

### ISSUE THREAD

**Issue by danieldietrich** [pahen/madge#258]
![Screenshot 2020-09-10 at 11 19 21](https://user-images.githubusercontent.com/743833/92710764-5bbce880-f358-11ea-83fd-2e4a6ad6e37e.png)

There are other files that use wildcard exports, like `comp/index.ts`:

```ts
export { comp } from './comp'
export { assign, render } from './render'
export * from './types'
```

The dependency between `comp/index.ts` and `comp/types.ts` is rendered correctly.

So why it does not happen for the root `index.ts`? 🤔

**Comment by johnnaegle** [comment]
I think this may be fixed by a typescript upgrade in the project.

I added a failing test in my fork for a similar case:

When the aggregation is this:
```typescript
import { nombre } from "./c-named";
export * as default from "./a-default";
export { named } from "./b-named";
```

That fails with this error:
```
 typescript-eslint:typescript-estree:createSourceFile Getting AST without type information in TS mode for: /Users/jnaegle/code/my-madge/test/typescript/aggregate/estree.ts +0ms
  tree error getting dependencies: Identifier expected. 'default' is a reserved word that cannot be used here. +9ms
  tree TSError: Identifier expected. 'default' is a reserved word that cannot be used here.
```

I believe this was fixed in Typescript 4.1, see here: https://github.com/microsoft/TypeScript/pull/39803

**Comment by PabloLION** [comment]
Thanks for the suggestion. Now typescript is added as an optional peer dependency in #350 so this problem should be solved.
